lib/src/constants.dart

```diff
@@ -5,6 +5,10 @@ final RegExp importExportDartPackageRegex =
 /// Regex used to detect all Sass import directives.
 final RegExp importScssPackageRegex = RegExp(r'''\@import\s+['"]{1,3}package:\s*([a-zA-Z0-9_]+)\/[^;]+''');
 
+/// Regex used to detect all Less import directives.
+final RegExp importLessPackageRegex =
+    RegExp(r'@import[\t ]+(?:\(.*\)\s+)?"(?:packages\/|package:\/\/)([a-zA-Z1-9_-]+)\/');
```

Comment:
Is there a reason to use `[\t ]+` after `@import` instead of `\s+`?

Reply:
I'm not familiar with LESS so I'll defer to @Anrock

Reply:
I'm not really familiar with less either, but from the spec it seemed like no newlines were allowed after `@import` and `s` includes newlines. So I've used `[\t ]` to allow only non-linebreaking whitespace.

Reply:
My biggest concern here was other whitespace (non-breaking space or zero-width, for example), but playing with http://lesscss.org/less-preview/ it does look like newline is allowed.